src/index.js

```diff
@@ -377,26 +376,46 @@ function createConfig(options, entry, format, writeMeta) {
 							},
 						}),
 					!useTypescript && flow({ all: true, pretty: true }),
-					// Only used for async await
-					babel({
-						// We mainly use bublé to transpile JS and only use babel to
-						// transpile down `async/await`. To prevent conflicts with user
-						// supplied configurations we set this option to false. Note
-						// that we never supported using custom babel configs anyway.
-						babelrc: false,
-						exclude: 'node_modules/**',
-						plugins: [
-							require.resolve('@babel/plugin-syntax-jsx'),
-							[
-								require.resolve('babel-plugin-transform-async-to-promises'),
-								{ inlineHelpers: true, externalHelpers: true },
-							],
-							[
-								require.resolve('@babel/plugin-proposal-class-properties'),
-								{ loose: true },
-							],
-						],
-					}),
+					babel(
+						(await isFile('.babelrc')) || (await isFile('.babelrc.js'))
+							? {
+									// Babelrc exists, allow project to configure babel
+									exclude: 'node_modules/**',
```

Comment:
not sure if that exclude is needed, babel@7 wont try to transpile things in node_modules anyway (without explicitly stating it do so)